### DISCUSSION TITLE
Need some help with this firmware

### DISCUSSION THREAD

**Discussion by Chemdawgds** [Jyers/Marlin#1643] [Q&A]
Hello.  I found that this firmware :https://github.com/Jyers/Marlin/releases/download/v2.0.1/E3V2-BLTouch-5x5-v4.2.2-v2.0.1.bin   is exactly what I want.  However, I need to change the bed size to 300X300.  

I'm finding it rather difficult to edit the source and compile myself.  I cannot seem to get everything right.  Is there anyway  someone could take this exact firmware and make the bed size 300X300?

**Comment by vhristenco** [comment]
[Here](https://drive.google.com/file/d/1ef_4f4_X_ln4kGVhySYF09e5Z2lF1wjf/view?usp=sharing)

**Reply by Tapap84** [reply]
that was nice of ya
